.github/workflows/_mac-test.yml

```diff
@@ -80,6 +80,11 @@ jobs:
         run: |
           sysctl machdep.cpu.brand_string kern.osproductversion
 
+      - name: Install build toolchain
+        run: |
+          brew update --quiet
+          brew install --formula cmake ninja
```

Comment:
Why?

Reply:
Yeah let's just not do things like this, the mac runners we use are not ephemeral so changes like this affect everyone else.

Reply:
Both `cmake` and `ninja` can be `pip`-installed instead, which should make sure the installs get cleaned up like all other dependencies in a CI job.

Reply:
Reverted. Trying to figure out an alternative to resolve:

- https://github.com/pytorch/pytorch/issues/158728#issuecomment-3095344917
- pytorch/vision#8421